### ISSUE TITLE
Small typo in Agenda setup (II)

### ISSUE THREAD

**Issue by dianrob** [rougier/emacs-gtd#14]
Agenda setup (II)

It’s now time to work on custom agenda view that will display meetings for the day, task that need to be done and deadlines. **We’all** also add the content of the inbox **ad** a reminder to file entries, and the tasks we’ve completed today. So overall, our custom agenda headers will be:

p.s.: Thanks for the great tutorial.

**Comment by rougier** [comment]
Thanks for the correction. Can you make a PR?

**Comment by dianrob** [comment]
I am new to github, sorry for the late reply. PR stands for pull request?

**Comment by rougier** [comment]
Yes, exactly. Sorry for the confusion. You can a PR by forking the repository and edit the file from the web interface (in your forked repository) and GitHub will propose you to make a PR that I can merge. I could of course doing it myself based on your report, but if you make a PR, your name will appear in the commits.
